src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAroundBasic.java

```diff
@@ -235,6 +237,12 @@ void forIterator()
         int i = (int) ( 2 / 3 );
         return null;
     }
+
+    void forColon() {
+        ArrayList<Integer> ll = new ArrayList<Integer>();
```

Comment:
Please use int[] instead of list to avoid unnecessary import and large diff in test

Reply:
done